# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/media/webcodecs/moz.build
# Commit: ca737b661c6e
# Full Hash: ca737b661c6eeaecfebf6f2bd27aca46ef656ae4
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-08-23 04:21:33
# Description:
#   Bug 1849271 - Flip test when checking for the presence of optional colorspace members on a VideoDecoderConfig. r=media-playback-reviewers,alwu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D186530
# ==============================================================================

diff -r bc91ede31572 -r ca737b661c6e dom/media/webcodecs/moz.build
--- a/dom/media/webcodecs/moz.build	Tue Aug 22 14:03:58 2023 +0000
+++ b/dom/media/webcodecs/moz.build	Tue Aug 22 14:33:21 2023 +0000
@@ -8,6 +8,7 @@
     BUG_COMPONENT = ("Core", "Audio/Video: Web Codecs")
 
 MOCHITEST_MANIFESTS += ["test/mochitest.ini"]
+CRASHTEST_MANIFESTS += ["crashtest/crashtests.list"]
 
 # For mozilla/layers/ImageBridgeChild.h
 LOCAL_INCLUDES += [